### PR TITLE
Task/pi 2476/generalize docs from ctv

### PR DESCRIPTION
**Pull request by rblaa**
first cut at an updated and more generalized version of the TAR docs

TODO: remove GETTING_STARTED.md files in ref apps, then should just point to the README.md here instead.

To review:
* best approach is to read thru the docs as they were meant to be displayed to the end user.
   * [README](https://github.com/socialvibe/truex-ad-renderer-web-integration/blob/task/PI-2476/generalize-docs-from-ctv/README.md)
   * [DOCS](https://github.com/socialvibe/truex-ad-renderer-web-integration/blob/task/PI-2476/generalize-docs-from-ctv/DOCS.md) 
* viewing the raw diffs is ok, but harder to see

Question: should i show a logo. It was showing the truex one, perhaps an Infillion one is better? 
I am hiding the logo for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,96 @@
...
-## Setup
-This document describes the initial steps needed to make use of the `TruexAdRenderer` in an HTML5 web application intended for Smart TVs and game consoles, i.e. for the so-called "10 foot" experience.
+The library provides a class called the `TruexAdRenderer` (TAR). TAR is supports running within any web content, 
+specifically for desktop and mobile web content, as well as for the "Connected TV" (CTV) platforms, such as Smart TVs
+and game consoles, i.e. for the so-called "10 foot" experience.
 
-The true[X] ad renderer is available as an `npm` module. For the typical web app based around a `package.json` project file, one adds the true[X] dependency as follows:
+## Setup
+The TrueX ad renderer is available as an `npm` module. For the typical web based development around a `package.json` 
+project file, one adds the TrueX dependency as follows:
```
Comment (jalbini) [inline README.md:12]: Please use second person voicing

File: README.md [hunk 2]
```diff
@@ -1,101 +1,96 @@
-# true[X]  Web Integrations
-Documentation and reference apps for integrating true[X]'s CTV web ad renderer
+# TruexAdRenderer Web Integration: Quick Guide
 
+This document describes the how to use Infillion's TrueX web ad renderer, a library that allows the presentation of
+interactive ads, intended for use in "Full Experience Playback" (FEP) scenarios such as video web sites or applications.
 
-## Setup
-This document describes the initial steps needed to make use of the `TruexAdRenderer` in an HTML5 web application intended for Smart TVs and game consoles, i.e. for the so-called "10 foot" experience.
+The library provides a class called the `TruexAdRenderer` (TAR). TAR is supports running within any web content,
```
Comment (jalbini) [inline README.md:6]:
```suggestion
The library provides a class called the `TruexAdRenderer` (TAR). TAR supports running within any web content, 
```

File: README.md [hunk 3]
```diff
@@ -1,101 +1,96 @@
...
-### Reference Apps
+To actually integrate the TrueX ad renderer, you have to create and invoke it during your app's video playback 
+when the time of an ad break is reached.
 
-1. [Generic Reference App](https://github.com/socialvibe/truex-ctv-web-reference-app)
-1. [Google Ad Manager: Server-side Ad Insertion Reference App](https://github.com/socialvibe/truex-ctv-web-google-ad-manager-reference-app)
-1. [Google Ad Manager: Client-side Ad Insertion Reference App](https://github.com/socialvibe/truex-ctv-google-ima-csai-ref-app)
-
-### Code Sample
+The following code provides an example of the style of how to integration to TAR, once a Truex ad has been detected
```
Comment (jalbini) [inline README.md:28]: I don't understand this sentence

File: README.md [hunk 4]
```diff
@@ -1,101 +1,96 @@
...
+interactive ads, intended for use in "Full Experience Playback" (FEP) scenarios such as video web sites or applications.
 
-## Setup
-This document describes the initial steps needed to make use of the `TruexAdRenderer` in an HTML5 web application intended for Smart TVs and game consoles, i.e. for the so-called "10 foot" experience.
+The library provides a class called the `TruexAdRenderer` (TAR). TAR is supports running within any web content, 
+specifically for desktop and mobile web content, as well as for the "Connected TV" (CTV) platforms, such as Smart TVs
+and game consoles, i.e. for the so-called "10 foot" experience.
 
-The true[X] ad renderer is available as an `npm` module. For the typical web app based around a `package.json` project file, one adds the true[X] dependency as follows:
+## Setup
```
Comment (jalbini) [inline README.md:10]: This section needs to include at least a mention that you can also just link the library, aka:

```javascript
 <script src="https://cdn.jsdelivr.net/npm/@truex/ctv-ad-renderer@1.9.0"></script>
```

File: DOCS.md [hunk 5]
```diff
@@ -92,117 +115,225 @@ This messaging will be displayed to the user for several seconds, after which th
 
 ## How to use TruexAdRenderer
 
-### When to show true\[X\]
+### Setup
```
Comment (jalbini) [inline DOCS.md:118]: Need to mention here that you can just a hosted library instead, a.k.a:

```javascript
 <script src="https://cdn.jsdelivr.net/npm/@truex/ctv-ad-renderer@latest"></script>
 ```

File: DOCS.md [hunk 6]
```diff
@@ -92,117 +115,225 @@ This messaging will be displayed to the user for several seconds, after which th
...
+Here we describe the essential methods of the `TruexAdRenderer` class, as well as ad events that can be listened for.
 
-### Installing true[X] Library
+Note that there are actually 3 possible classes to use:
+```javascript
+TruexAdRendererCTV
+TruexAdRendererDesktop
+TruexAdRendererMobile
+```
+One should use the appropriate one for your platform, so as to allow the correct ad tracking to be used.
```
Comment (jalbini) [inline DOCS.md:258]:
```suggestion
You should use the appropriate one for your platform, so as to allow the correct ad tracking to be used.
```

File: DOCS.md [hunk 7]
```diff
@@ -92,117 +115,225 @@ This messaging will be displayed to the user for several seconds, after which th
...
 
-The true[X] interactive ad component and its rendering logic are distributed as part of an npm package. It is required for the hosting app to include the renderer library in order for it to be used, this can be achieved by running this `npm` command:
+If one uses `TruexAdRenderer`, that maps to `TruexAdRendererCTV`
```
Comment (jalbini) [inline DOCS.md:260]: Is `TruexAdRenderer` deprecated? If so, you don't need to mention it here, just cut this line. It's confusing

**Review by jalbini — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (rblaa) [reply README.md:28]: How about:

Once a TrueX ad is detected, the renderer needs to be created and displayed. The following code provides an example of the typical approach of integrating to TAR.

File: DOCS.md (hunk 7 above)
Comment (rblaa) [reply DOCS.md:260]: It is not deprecated, and now it probably should be.
That tracking category thing is the reason, and it is a little annoying, e.g. fep_ctv vs fep vs fep_mobile
And it only matters for a second, only until the choice card code takes over

The thing is we are using that class currently in all our reference apps and docs.

I need to check that the webview bridge actually does the right thing.

File: README.md (hunk 4 above)
Comment (rblaa) [reply README.md:10]: done

File: README.md (hunk 3 above)
Comment (rblaa) [reply README.md:28]: fixed

File: DOCS.md (hunk 5 above)
Comment (rblaa) [reply DOCS.md:118]: done

File: README.md (hunk 2 above)
Comment (rblaa) [reply README.md:6]: fixed

File: README.md (hunk 1 above)
Comment (rblaa) [reply README.md:12]: fixed

File: DOCS.md (hunk 6 above)
Comment (rblaa) [reply DOCS.md]: done

File: DOCS.md (hunk 7 above)
Comment (rblaa) [reply DOCS.md]: it is deprecated now, which is now mentioned.

**Review by zachschrier — APPROVED** [review]
👍🏻 Looks good to me.

**Review by jalbini — APPROVED** [review]

**Pull request merged**